tests/wpunit/Tribe/AssetsTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace Tribe;
+
+use Tribe__Assets as Assets;
+use Tribe__Main as Plugin;
+
+class AssetsTest extends \Codeception\TestCase\WPTestCase {
+	/**
+	 * @test
+	 *
+	 * @since TBD
+	 */
+	public function it_should_have_translations() {
+		$locale = 'en';
```

Comment:
Since English is the default language, I would add a data provider for this test method and provide the language for it, say `en` and another `de`.
Scaffold the translation files for the second language (`de`) in my example and test those too.

While the fallback/default English logic might keep working, the logic for actual translation might be not down the road and I think this will protect the code a bit more from regression.

Reply:
I will add that I feel like it creates a bigger safety net, but the default language is not English, it's empty string. Looking for `en` is not the default I mean.

Reply:
Ah, thanks. I did not know that.
The strings are, by default in our code, in English and that could obscure a non-passing test.